### DISCUSSION TITLE
QIDI I-MATE S QUERY

### DISCUSSION THREAD

**Discussion by Andhan17** [QIDITECH/QIDISlicer#106] [General]
I recently purchased a QIDI I-MATE S and I am looking for the software for it to be able to use it any ideas on which code to use new to 3d printers. 
ID-  0038005c-3056500d-20343543-ff
version - v4.2.13.2   /w10.0.12
UI - Blue+1.1

**Comment by QIDITECH** [comment]
Please contact after-sales service for assistance.
